src/distributions/mod.rs

```diff
@@ -59,119 +65,41 @@
 //! documentation in the [`uniform`] module. Doing so enables generation of
 //! values of type `T` with  [`Rng::gen_range`].
 //!
-//!
-//! # Other distributions
+//! ## Open and half-open ranges
 //!
 //! There are surprisingly many ways to uniformly generate random floats. A
 //! range between 0 and 1 is standard, but the exact bounds (open vs closed)
 //! and accuracy differ. In addition to the [`Standard`] distribution Rand offers
 //! [`Open01`] and [`OpenClosed01`]. See "Floating point implementation" section of
 //! [`Standard`] documentation for more details.
 //!
-//! [`Alphanumeric`] is a simple distribution to sample random letters and
-//! numbers of the `char` type; in contrast [`Standard`] may sample any valid
-//! `char`.
-//!
-//! [`WeightedIndex`] can be used to do weighted sampling from a set of items,
-//! such as from an array.
-//!
-//! # Non-uniform probability distributions
-//!
-//! Rand currently provides the following probability distributions:
-//!
-//! - Related to real-valued quantities that grow linearly
-//!   (e.g. errors, offsets):
-//!   - [`Normal`] distribution, and [`StandardNormal`] as a primitive
-//!   - [`Cauchy`] distribution
-//! - Related to Bernoulli trials (yes/no events, with a given probability):
-//!   - [`Binomial`] distribution
-//!   - [`Bernoulli`] distribution, similar to [`Rng::gen_bool`].
-//! - Related to positive real-valued quantities that grow exponentially
-//!   (e.g. prices, incomes, populations):
-//!   - [`LogNormal`] distribution
-//! - Related to the occurrence of independent events at a given rate:
-//!   - [`Pareto`] distribution
-//!   - [`Poisson`] distribution
-//!   - [`Exp`]onential distribution, and [`Exp1`] as a primitive
-//!   - [`Weibull`] distribution
-//! - Gamma and derived distributions:
-//!   - [`Gamma`] distribution
-//!   - [`ChiSquared`] distribution
-//!   - [`StudentT`] distribution
-//!   - [`FisherF`] distribution
-//! - Triangular distribution:
-//!   - [`Beta`] distribution
-//!   - [`Triangular`] distribution
-//! - Multivariate probability distributions
-//!   - [`Dirichlet`] distribution
-//!   - [`UnitSphereSurface`] distribution
-//!   - [`UnitCircle`] distribution
```

Comment:
It would be nice to move this text to `rand_distr`.

Reply:
I think it did get moved?

Reply:
You are right, it's in d4221c60b79405a344b40aee7c318741d69e23a8.